package.json

```diff
@@ -23,6 +23,12 @@
     "prepare": "husky install",
     "postinstall": "node node_modules/puppeteer/install.js"
   },
+  "bin": {   
+    "autocannon-ui": "index.js"
+  },
+  "overrides": {
+    "puppeteer-core": "^18.2.1"
+  },
```

Comment:
do we still need this?

Reply:
We do need it if we want to keep using the chrome-aws-lambda package. I am checking if we can remove it or not